builds/e2e/templates/nested-parent-vm-setup.yaml

```diff
@@ -1,20 +1,24 @@
+parameters:
+  transportType: 'amqp'
+
 jobs:
-  - job: SetupVM_level5
-    displayName: SettingUp level 5
+  - job: SetupVM_level5_${{ parameters.transportType }}
+    displayName: SettingUp level 5 for ${{ parameters.transportType }} 
     timeoutInMinutes: 180
     variables:
       artifactName: iotedged-ubuntu18.04-amd64
       identityServiceArtifactName: packages_ubuntu-18.04_amd64
       identityServicePackageFilter: aziot-identity-service_*_amd64.deb
       vsts.project: $(System.TeamProjectId)
       level: 5
-      deploymentFile: 'nestededge_topLayerBaseDeployment.json'
+      deploymentFile: 'nestededge_topLayerBaseDeployment_${{ parameters.transportType }}.json'
       parentName: ''
       parentDeviceId: ''
+      transportType: ${{ parameters.transportType }}
     pool:
      name: $(pool.name)
      demands:
-       - agent-group -equals $(agent.group)
+       - agent-group -equals $(agent.group)-${{ parameters.transportType }}
        - Agent.OS -equals Linux
        - Agent.OSArchitecture -equals X64
        - run-idle -equals true
```

Comment:
1. `- run-idle -equals true` should no longer exist. It was meant to exist as my prove-of-concept for a locking scheme. Looks like the code got copied before I clean them up. Please remove. 
2. Please add a remark regarding `$(agent.group)-${{ parameters.transportType }}`'s vnet. Where are they? what VNET value?